library/src/main/java/org/apache/apex/malhar/lib/dedup/BoundedDedupOperator.java

```diff
@@ -156,8 +156,7 @@ protected Slice getKey(Object tuple)
   protected void putManagedState(Object tuple)
   {
     Slice key = getKey(tuple);
-    ((ManagedTimeStateImpl)managedState).put(getBucketId(key), DEFAULT_CONSTANT_TIME,
-        key, new Slice(new byte[0]));
+    ((ManagedTimeStateImpl)managedState).put(getBucketId(key), DEFAULT_CONSTANT_TIME, key, new Slice(null, 0, 0));
```

Comment:
The changes related to Slice are part of different commit (APEXMALHAR-2492) which need to be merged before this PR.